cartoframes/viz/helpers/utils.py

```diff
@@ -0,0 +1,5 @@
+
+def serialize_palette(palette):
+    if isinstance(palette, list):
```

Comment:
Adding `tuple` to this might be a good idea.
`isinstance(palette, (list, tuple))`